lib/msf/core/exploit/remote/kerberos/ticket.rb

```diff
@@ -63,14 +63,9 @@ def forge_ticket(enc_key:, enc_type:, start_time:, end_time:, sname:, flags:,
             ccache = ticket_as_krb5ccache(ticket, opts: opts)
 
             if save_ccache
-              info = []
-              info << "realm: #{opts[:realm].upcase}"
-              info << "serviceName: #{opts[:server].to_s.downcase}"
-              info << "username: #{opts[:client].to_s.downcase}"
-
-              path = store_loot('mit.kerberos.ccache', 'application/octet-stream', domain, ccache.encode, nil, info.join(', '))
-              print_good("MIT Credential Cache ticket saved on #{path}")
+              Kerberos::Ticket::Storage.store_ccache(ccache, host: domain, framework_module: self)
```

Comment:
```suggestion
              Kerberos::Ticket::Storage.store_ccache(ccache, framework_module: self)
```

In this context we only have the domain - and we won't always be able to resolve that to a host, and the dns lookup takes a while to fail on macosx. I know you're just updating the existing pattern here, but we might as well remove it whilst we're here 🎉